Security in WordPress/Security.md

```diff
@@ -187,5 +187,42 @@ _______________________________________________________________
 [+] Requests made: 61
 [+] Memory used: 35.848 MB
 ```
+## Code snippets
```

Comment:
Maybe rename this to 

## Useful hardening code snippets

Or something similar?

Reply:
Makes sense, 👍